_1327/documents/templates/editor.html

```diff
@@ -2,231 +2,305 @@
 {% load i18n %}
```

Comment:
That's not what I asked. Please separate it into multiple commits so someone else will find the actual change as well.